### PR TITLE
REL-552: Support of ITC imaging and Altair for GNIRS

### PR DESCRIPTION
**Pull request by olesjasmirnova**
Integrated GNIRS ITC imaging with the OT. Also now GNIRS ITC supports Altair.

### CODE REVIEW

File: app/spdb/build.sbt [hunk 1]
```diff
@@ -233,16 +233,16 @@ def fnussber(version: Version) = AppConfig(
     "-XX:MaxPermSize=196M",
     "-Dedu.gemini.site=north",
     "-Dcron.*.edu.gemini.dbTools.html.ftpHost=localhost",
-    "-Dcron.*.edu.gemini.dbTools.html.ftpDestDir=/Users/fnussber/.spdb/sftp",
+    "-Dcron.*.edu.gemini.dbTools.html.ftpDestDir=/Users/osmirnov/.spdb/sftp",
```
Comment (cquiroz) [inline app/spdb/build.sbt]: I deleted florian's spdb profile, It is better if you add your own profile with your name (osmirnov). Remember to add it to the list on line 25. If this is complicated I can do it for you when I merge my  PR #941

**Comment by olesjasmirnova** [comment]
Hi Carlos, 
I would appreciate a lot if you could do it for me, since I have no idea how to do this... Thank you!

File: bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/gnirs/EdCompInstGNIRS.java [hunk 2]
```diff
@@ -76,6 +76,19 @@ public EdCompInstGNIRS() {
         _w.disperser.setChoices(SpTypeUtil.getFormattedDisplayValueAndDescriptions(Disperser.class));
         _w.centralWavelength.setChoices(_getDefaultWavelengths());
 
+        //TODO: Add filter and acqMirror to GNIRS instrument component
```
Comment (cquiroz) [inline bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/gnirs/EdCompInstGNIRS.java]: If you want  you can just deleted the commented code

**Comment by cquiroz** [comment]
Code-wise this looks very good. I can't comment about the calculations themselves but I guess you have that part clear

File: bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/gnirs/EdCompInstGNIRS.java (hunk 2 above)
Comment (olesjasmirnova) [inline bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/gnirs/EdCompInstGNIRS.java]: In some places I commented out some code which wasn't ready for this merge yet, but I'd like to use it later. So if it's ok, I'd prefer to leave it there...

Comment (cquiroz) [inline bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/gnirs/EdCompInstGNIRS.java]: That's fine 👍

**Comment by olesjasmirnova** [comment]
Thank you! I checked the calculations, but definetely it requires more testing to be done by other people, so it would be nice if this would could available for testing this week...
Do you think this be included in today's test release, or was I too late for that?

**Comment by cquiroz** [comment]
Sorry, the release went out, but I'll merge this right now

**Pull request merged**